ecopod/sendRawTransactionConditional.md

```diff
@@ -0,0 +1,53 @@
+# Purpose
+
+To fully unlock ERC-4337 on the OP-Stack.
+
+# Summary
+
+By providing an auxilliary transaction submission mechanism, [eth_sendRawTransactionConditional](https://notes.ethereum.org/@yoav/SkaX2lS9j), we can enable Bundlers to submit Entrypoint transactions with stronger guarantees, avoiding costly inadvertent reverts.
+
+We can implement this endpoint in op-geth with an additional layer of external validation to ensure this endpoint is safe from DoS attacks, does not supersede `eth_sendRawTransaction`, and can be deprecated if native account abstraction efforts in the future absolves the need for this endpoint.
```

Comment:
I recommend adding a link to the following document which is the current proposed AA roadmap: https://notes.ethereum.org/@yoav/AA-roadmap-May-2024